build.gradle.kts

```diff
@@ -93,20 +82,26 @@ spotless {
         // due to the bug: https://github.com/diffplug/spotless/issues/532
         licenseHeaderFile("spotless.license.java") // contains '$YEAR' placeholder
 
-        targetExclude("**/generated/**/*.java")
+        targetExclude("{${layout.buildDirectory}}/generated/sources/buildConfig/**/*.java")
```

Comment:
I think it's safe to say that we don't want `spotless` to act on any generated code we'll ever have, not just one in `buildConfig`.